src/postgres/tree/PostgresEnterCredentialsTreeItem.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureTreeItem, ISubscriptionContext } from "vscode-azureextensionui";
+import { localize } from "../../utils/localize";
+import { PostgresDatabaseTreeItem } from "./PostgresDatabaseTreeItem";
+
+export class PostgresEnterCredentialsTreeItem extends AzureTreeItem<ISubscriptionContext> {
```

Comment:
I don't think you'd need a whole treeItem class for this.  For connecting to Github, I used a generic treeItem like so: https://github.com/microsoft/vscode-azuretools/blob/e8e7cf2b8733f6b6ffc380a01aa2996a260caf3c/appservice/src/tree/DeploymentsTreeItem.ts#L84-L89